ivadomed/main.py

```diff
@@ -413,6 +471,8 @@ def cmd_train(context):
                       mt_metrics.specificity_score,
                       mt_metrics.intersection_over_union,
                       mt_metrics.accuracy_score]
+        if context["unet_3D"]:
+            metric_fns.remove(mt_metrics.hausdorff_score)
```

Comment:
Just realise --> this block could certainly be moved out of the training loop I think.